src/main/java/core/basesyntax/FigureSuplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.lang.invoke.SwitchPoint;
+import java.util.Random;
+
+public class FigureSuplier {
```

Comment:
The class name `FigureSuplier` is misspelled. It should be `FigureSupplier` to maintain consistency with the naming convention used elsewhere.